### DISCUSSION TITLE
Welcome to PS3-Quick-Disc-Decryptor Discussions!

### DISCUSSION THREAD

**Discussion by ElektroStudios** [ElektroStudios/PS3-Quick-Disc-Decryptor#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wibi9424** [comment]
is it can decrypt pkg file?

**Reply by ElektroStudios** [reply]
> is it can decrypt pkg file?

Hi. **PS3 Quick Disc Decryptor**, as its name suggests, is used to decrypt PS3 disc images (ISO files).

To decrypt (read: use, activate) a pkg file in an emulator or console, you would need the corresponding RAP (license key) file.

I've uploaded a custom, huge RAP file collection in archive.org which you can find through [this](https://archive.org/details/ps3-rap-collection-2024) link. You can also look at [nopaystation](https://nopaystation.com/browse) for a specific pkg / DLC title.

Cheers!
